websites/Z/Zybooks/presence.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable one-var */
+/* eslint-disable no-one-time-vars/no-one-time-vars */
+const presence = new Presence({
+	clientId: "1152294065115779102",
+});
+
+function getTextContent(querySelectorString: string): string {
+	return (
+		(
+			document.querySelector(querySelectorString) as HTMLElement | null
+		)?.textContent?.trim() || ""
+	);
+}
+
+const zybooksTimestamp: {
+	hash: string;
+	timestamp: number;
+} = {
+	hash: "",
+	timestamp: Date.now(),
+};
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = document.location;
+	let hash: string = href;
+	const presenceData: PresenceData = {
+		startTimestamp: zybooksTimestamp.timestamp,
+		largeImageKey: "https://i.imgur.com/I4RTS73.png",
```

Comment:
The logo should be a 512x512 square.

Reply:
The logo isn't square to begin with so I don't know what to do

Reply:
There are tools that can edit images and do things like upscaling them, squaring them, etc.

Reply:
> The logo isn't square to begin with so I don't know what to do

For reference the way I do it is:
1- Download the image
2- Open it in paint.net
3- Change canvas size to 512x512 and use the image tool to place the old image in the middle
4- Delete the white parts
5- _boom_ you're doine